modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -1,54 +1,54 @@
 // This is included in the following assemblies:
 //
-// ipi-install-expanding-the-cluster.adoc
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
 
 [id='preparing-the-bare-metal-node_{context}']
-
 = Preparing the bare metal node
 
 Expanding the cluster requires a DHCP server. Each node must have a DHCP reservation.
 
-ifeval::[{product-version}>4.6]
 [IMPORTANT]
 .Reserving IP addresses so they become static IP addresses
 ====
-Some administrators prefer to use static IP addresses so that each node's IP address remains constant in the absence of a DHCP server. To use static IP addresses in the {product-title} cluster, *reserve the IP addresses in the DHCP server with an infinite lease*. After the installer provisions the node successfully, the dispatcher script will check the node's network configuration. If the dispatcher script finds that the network configuration contains a DHCP infinite lease, it will recreate the connection as a static IP connection using the IP address from the DHCP infinite lease. NICs without DHCP infinite leases will remain unmodified.
+Some administrators prefer to use static IP addresses so that each node's IP address remains constant in the absence of a DHCP server. To configure static IP addresses with NMState, see "(Optional) Configuring host network interfaces in the `install-config.yaml` file" in the "Setting up the environment for an OpenShift installation" section for additional details.
```

Comment:
The first sentence is a bit weird when combined with this paragraph, because the two paragraphs are describing two different problems:

1. You don't have a DHCP server
2. You have a DHCP server, but once a node has an address you don't want to rely on it always being around to keep the same address.

And this first sentence is about the second one. It was somewhat confusing before, but doubly so now.

Reply:
I can omit the old approach, since using NMState is preferred now. Do you think that's a better approach?

Reply:
Probably @cybertron would be a better person to answer that.

Reply:
I think it's fair to remove this. We aren't going to want people to use the infinite lease functionality now that there's a better option for static IP configuration. Anyone deploying an older release that still needs this can go look at that version's docs.

Reply:
Done.

Reply:
Funny, I just created a PR based on recommendation of https://bugzilla.redhat.com/show_bug.cgi?id=2053972. PR is https://github.com/openshift/openshift-docs/pull/42152.  I wonder if my line "Setting IP addresses with an infinite lease is incompatible with network configuration deployed by using the Machine Config Operator." is still needed

Reply:
It would be good to have a backportable change since the MCO conflict is more significant in older versions. It would be good to both remove the section from 4.10 and up but also add the note to 4.9 and older.